.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: php
+
+php:
+  - '7.1'
```

Comment:
I only want to support 7.2 and 7.3 for now, I'll be updating the composer.json to reflect this